JotunnLib/Managers/DungeonManager.cs

```diff
@@ -251,7 +256,7 @@ private void OnZoneSystemSetupLocations()
 
                     var env = kvp.Value;
                     env.FixReferences(true);
-                    UnityEngine.Object.Instantiate<GameObject>(env, DungeonRoomContainer.transform);
+                    UnityEngine.Object.Instantiate<GameObject>(env); // Intentionally omitting DungeonRoomContainer.transform
```

Comment:
The env gameobject needs to be enabled in order for the component awakes to run.